### DISCUSSION TITLE
Welcome to RVCE-3rd-Sem-CSE-lab-programs-OS-DSA Discussions!

### DISCUSSION THREAD

**Discussion by 18praneeth** [18praneeth/RVCE-3rd-Sem-CSE-lab-programs-OS-DSA#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 18praneeth** [comment]
Hey glad to know it's helpful, I have not taken down any code.
You can access it here: https://github.com/18praneeth/RVCE-3rd-Sem-CSE-lab-programs-OS-DSA/tree/main/Operating%20System
It's not organized properly, but you can refer to both set1 and set2, it has slight modifications in some programs.

**Reply by Suharsha13kk** [reply]
I HAD JUST REALISEDDD 
I'm sorry, I didn't go through it properly. 

Also, thank youuuu for making this available on github

**Reply by 18praneeth** [reply]
Welcome.
All the best for your exams
